### PR TITLE
interfaces/i2c: allow modifying device-specific sysfs entries

### PR DESCRIPTION
**Pull request by jdstrand**
Fix for https://forum.snapcraft.io/t/how-can-i-declare-i2c-slot/364

### CODE REVIEW

**Comment by jdstrand** [comment]
- linode:ubuntu-core-16-64:tests/main/failover:emptyinitrd

This is an unrelated test failure.

**Comment by jdstrand** [comment]
The yakkety failure is unrelated: FAIL: overlord_test.go:360: overlordSuite.TestEnsureLoopPrune

File: interfaces/builtin/i2c.go [hunk 1]
```diff
@@ -92,6 +92,7 @@ func (iface *I2cInterface) AppArmorConnectedPlug(spec *apparmor.Specification, p
 
 	cleanedPath := filepath.Clean(path)
 	spec.AddSnippet(fmt.Sprintf("%s rw,", cleanedPath))
+	spec.AddSnippet(fmt.Sprintf("/sys/devices/platform/**.i2c/%s/** rw,", strings.TrimPrefix(path, "/dev/")))
```
Comment (niemeyer) [inline interfaces/builtin/i2c.go]: Won't this hand off access to every single device in the system instead of just the specified one?

**Comment by niemeyer** [comment]
Sorry, nevermind. I'm blind!

**Pull request merged**